### PR TITLE
Upgrade rrweb fork

### PR DESCRIPTION
**Pull request by timgl**
## Changes

Pulling from git didn't work so I published our own version of rrweb.

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10418,6 +10429,11 @@ rrweb-snapshot@^1.0.3:
   resolved "https://registry.yarnpkg.com/rrweb-snapshot/-/rrweb-snapshot-1.0.6.tgz#2b1143d2e8b5bba54c54bd63e485cfc253763b8e"
   integrity sha512-ctwaNjFnomNFe446gOcMusSzIxLUnF0kq6ev0iuKa1W1lhiVcBoRdfpE8zDDRKee2MxHxt0rq2T5DMKSIAD6fQ==
 
+rrweb-snapshot@^1.1.2:
+  version "1.1.3"
+  resolved "https://registry.yarnpkg.com/rrweb-snapshot/-/rrweb-snapshot-1.1.3.tgz#a2b35dd481281e70a8f25972c77ea03930ae338a"
+  integrity sha512-FAMn6bonk1XCeoe87F9DxVpgeeSFCYoziLlzRI56S7Jb+t4lxX7LIN/VOqS3ZbVdGTiQLP4cosDtYU8pu6bnHg==
+
 rrweb@^0.9.14:
```
Comment (neilkakkar) [inline yarn.lock:10437]: wonder if this should still be here?

**Review by neilkakkar — APPROVED** [review]

**Comment by timgl** [comment]
ok so this still isn't working, our react-rrweb package still requires rrweb.

**Comment by neilkakkar** [comment]
I think that should be fine? (~~it's annoying and hacky, but fine?~~) - since we control that too? (and can modify https://github.com/PostHog/posthog-react-rrweb-player/blob/master/package.json#L70 ? ) - or am I missing something?

**Comment by neilkakkar** [comment]
At the same time, we _could_ work with two different versions (this is annoying and hacky) - and might lead to more problems if there's differences in encoding / decoding between the two different versions. As long as the one for recording is the new version, all is well?

**Comment by timgl** [comment]
Have now update that package to use @posthog/rrweb.

**Pull request merged**